src/components/Tasks/TasksTable.js

```diff
@@ -117,8 +118,8 @@ const TaskTable = ({ tasks }) => {
       fields={['Task', 'Dispute', 'Priority', 'Assigned to juror', 'Due date']}
       entries={sortedTasks}
       renderEntry={({ taskName, disputeId, priority, juror, dueDate }) => {
-        const formattedDate = dayjs(dueDate).format('YYYY-MM-DDTHH:mm:ssZ')
-        const hoursAndSec = dayjs(dueDate).format('HH:mm')
+        const formattedDate = dateFormat(dueDate, 'YYYY-MM-DDTHH:mm:ssZ')
```

Comment:
Ah sorry, what I meant is that we could store the formats themselves (`HH:mm`, `YYYY-MM-DDTHH:mm:ssZ`) in the same file with aliases (e.g. `hours-minutes` for `HH:mm`, `full` for `YYYY-MM-DDTHH:mm:ssZ`, etc.), so we can ensure they stay consistent across the app. But we can do this later, we don’t have that many at the moment :-)

Reply:
Resolving this for now!